go/go-introduction/if-statements/if-else.md

```diff
@@ -45,8 +50,7 @@ if x := 1 ; x > 0 {
 // 1 is positive
 ```
 
-Now, we would have only one output.
-
+This won't have any errors because the `else` is a part of the `if` statement.
```

Comment:
btw removing the "a" might be more appropriate here:

https://ell.stackexchange.com/questions/224909/difference-between-part-and-a-part